### PR TITLE
fix(fx-2407): adjusts show2 install shots query #trivial

### PR DESCRIPTION
**Pull request by dzucconi**
The type of this PR is: Bugfix

This PR resolves [FX-2407]

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[FX-2407]: https://artsyproduct.atlassian.net/browse/FX-2407

### CODE REVIEW

File: src/lib/Scenes/Show2/Components/Show2InstallShots.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export const Show2InstallShotsFragmentContainer = createFragmentContainer(Show2I
   show: graphql`
     fragment Show2InstallShots_show on Show {
       name
-      images {
+      images(default: false) {
```
Comment (dblandin) [inline src/lib/Scenes/Show2/Components/Show2InstallShots.tsx:59]: Wasn't sure what this argument did so I quickly looked it up:

https://github.com/artsy/metaphysics/blob/0b4ebf1dd09e6a515f8294dd0e480c2465e65a4a/src/schema/v2/show.ts#L355-L358

Looks like it determines whether default/cover images are included in the response set which makes sense 👍

**Review by dblandin — APPROVED** [review]
LGTM!

File: src/lib/Scenes/Show2/Components/Show2InstallShots.tsx (hunk 1 above)
Comment (dzucconi) [reply src/lib/Scenes/Show2/Components/Show2InstallShots.tsx:59]: Yeah, two things about this interface:

* `default` is a very bad name for this argument
* `coverImage` should be it's own distinct field probably, separate from installation views

It's probably a decent idea to deprecate the `images` field and build `installationViews` and `coverImage` as their own fields.

Comment (dblandin) [reply src/lib/Scenes/Show2/Components/Show2InstallShots.tsx:59]: Agreed on both points!

Deprecating the existing field and moving to separate interfaces sounds good. Maybe `coverImage` and `installationImages`? We'd still be modeling images, right?

**Pull request merged**